docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,467 @@
+      "text/plain": [
+       "<Figure size 701.723x264.88 with 1 Axes>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit_nature.circuit.library import SlaterDeterminant\n",
+    "\n",
+    "occupied_orbitals = (0, 2)\n",
+    "eig = np.sum(orbital_energies[list(occupied_orbitals)]) + transformed_constant\n",
+    "print(\"Eigenvalue:\")\n",
+    "print(eig)\n",
+    "\n",
+    "circuit = SlaterDeterminant(transformation_matrix[list(occupied_orbitals)])\n",
+    "circuit.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "bc169d46-52c0-4852-a6db-73c0e56d37f6",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<h3>Version Information</h3><table><tr><th>Qiskit Software</th><th>Version</th></tr><tr><td><code>qiskit-terra</code></td><td>0.21.0.dev0+04807a1</td></tr><tr><td><code>qiskit-nature</code></td><td>0.4.0</td></tr><tr><th>System information</th></tr><tr><td>Python version</td><td>3.9.6</td></tr><tr><td>Python compiler</td><td>GCC 8.5.0 20210514 (Red Hat 8.5.0-3)</td></tr><tr><td>Python build</td><td>default, Aug 11 2021 06:39:25</td></tr><tr><td>OS</td><td>Linux</td></tr><tr><td>CPUs</td><td>4</td></tr><tr><td>Memory (Gb)</td><td>15.31484603881836</td></tr><tr><td colspan='2'>Wed May 04 16:12:13 2022 EDT</td></tr></table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div style='width: 100%; background-color:#d5d9e0;padding-left: 10px; padding-bottom: 10px; padding-right: 10px; padding-top: 5px'><h3>This code is a part of Qiskit</h3><p>&copy; Copyright IBM 2017, 2022.</p><p>This code is licensed under the Apache License, Version 2.0. You may<br>obtain a copy of this license in the LICENSE.txt file in the root directory<br> of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.<p>Any modifications or derivative works of this code must retain this<br>copyright notice, and modified files need to carry a notice indicating<br>that they have been altered from the originals.</p></div>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import qiskit.tools.jupyter\n",
+    "\n",
+    "%qiskit_version_table\n",
+    "%qiskit_copyright"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "qiskit-nature",
+   "language": "python",
+   "name": "qiskit-nature"
```

Comment:
```suggestion
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
```
This should fix the kernel issue. @manoelmarques might have more insight on how to deal with this in a more general setting when he is back